drush/Commands/cgov/CgovUserCommands.php

```diff
@@ -90,6 +91,42 @@ public function blockAdminUser() {
 
   }
 
+  /**
+   * UnBlock Admin User.
+   *
+   * @command cgov:unblock-admin-user
+   * @aliases unblock-admin-user, ublkadm
+   * @usage drush cgov:unblock-admin-user
+   *   UnBlock the admin user (user 1).
+   * @bootstrap full
+   */
+  public function unblockAdminUser() {
+    /** @var \Drupal\user\Entity\User $account */
+    if ($account = User::load(1)) {
+      $account->activate();
+      $account->save();
+      if (\Drupal::moduleHandler()->moduleExists('samlauth')) {
+        // Unlink saml for admin user.
+        $authname = \Drupal::database()->select('authmap', 'm')
```

Comment:
It doesn't hurt to check first, but there's also no particular need to, is there? We could just issue the delete command and it will delete if anything exists.